tests/system/Config/fixtures/.env

```
@@ -4,6 +4,11 @@ SPACED="with spaces"
 
 NULL=
 
+char.expo.foo="exported foo"
+character.export.var=variable
+export char.var=character
+export char.exports=imports
+
```

Can you add a test for this spacing?
```suggestion
fruit.export = "banana"
```